clients/src/main/java/org/apache/kafka/common/requests/PushTelemetryRequest.java

```diff
@@ -83,8 +94,38 @@ public Uuid getClientInstanceId() {
         return this.data.clientInstanceId();
     }
 
+    public int getSubscriptionId() {
+        return this.data.subscriptionId();
+    }
+
+    public boolean isClientTerminating() {
+        return this.data.terminating();
+    }
+
+    public String getMetricsContentType() {
+        // Future versions of PushTelemetryRequest and GetTelemetrySubscriptionsRequest may include a content-type
+        // field to allow for updated OTLP format versions (or additional formats), but this field is currently not
+        // included since only one format is specified in the current proposal of the kip-714
+        return "OTLP";
```

Comment:
Should the return value be the same as what `ClientTelemetryPayload.contentType()` returns?

Reply:
We take the value from this method of the pushRequest and the populate the ClilentTelemetryPayload object and then pass it as an argument to the exportMetrics method.